plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -658,6 +658,12 @@ public static Optional<Snapshot> firstSnapshotAfter(Table table, long baseSnapsh
         checkArgument(current.snapshotId() != baseSnapshotId, "No snapshot after %s in %s, current snapshot is %s", baseSnapshotId, table, current);
 
         while (true) {
+            if (current.parentId() == null) {
```

Comment:
The `checkArgument` below is now redundant

Reply:
thanks, removed